src/ecalc/libraries/libecalc/common/libecalc/common/time_utils.py

```diff
@@ -52,19 +52,37 @@ class Periods:
     periods: List[Period]
 
     @classmethod
-    def create_periods(cls, times: List[datetime]) -> Periods:
+    def create_periods(cls, times: List[datetime], include_before: bool = True, include_after: bool = True) -> Periods:
```

Comment:
This will potentially fail with previous functionality, that defaulted to `False` for both `include_before` and `include_after`. Isnt it better to default to `False` then? Or do you have overview of all usage and that it will still work (or work better?) with the new functionality?

Reply:
maybe time to ask if we can enforce start and end time for ecalc models? ...

Reply:
Default was True, right? So I played it safe. That's what I tried to do at least

Reply:
heh...good question...without tests its harder to say ...it was a bit hard to read the diff...as usual :p..but yes...you are right, then we agree :)